Added service discovery tests

bmdepesa opened the pull request:
@sangeethah

sangeethah commented on tests/v3_api/common.py:
```diff
@@ -452,6 +452,22 @@ def validate_cluster(client, cluster, intermediate_state="provisioning",
+
+    cmd = 'ping -c 1 -W 1 {0}'.format(host)
+    output = kubectl_pod_exec(pod, cmd)
+    assert "0% packet loss" in str(output)
```
@bmdepesa You should be able check additionally that ping response has one of the entries from expected

bmdepesa commented:
@sangeethah please review

Re: cross-namespace validation, added both pods from setup and changed `create_and_validate_dns_record()` to check resolution from both pods

sangeethah commented:
LGTM
